students/tim_lurvey/lesson03/assignment/basic_operations.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/env/bin python
+""" Documentation for {file}
+This file contains basic operations for interaction with an sqlite3 database""".format(file=__file__)
+
+import logging
+from peewee import *
+from database_models import *
+
+logging.basicConfig(level=logging.ERROR)
+logger = logging.getLogger(__name__)
+
+
+def add_customer(customer_id, name, lastname, home_address, phone_number, email_address, status, credit_limit):
+    """This function will add a new customer to the sqlite3 database."""
+    try:
+        with database.transaction():
+            new_customer = Customer.create(
+                customer_id=customer_id,
+                name=name,
+                lastname=lastname,
+                home_address=home_address,
+                phone_number=phone_number,
+                email_address=email_address,
+                status=status,
+                credit_limit=credit_limit,
+            )
+            new_customer.save()
+            logger.info(f'Customer record created: {customer_id}')
+
+    except Exception as e:
```

Comment:
Please replace general exceptions with explicit exceptions. The problem with using general exceptions in released code is that they will captured everything, exceptions you have already seen and the ones you haven't, which may cause a new type of issue to go undetected or have unintended consequences because of improper handling.